response_operations_ui/controllers/case_controller.py

```diff
@@ -83,10 +83,10 @@ def get_case_groups_by_business_party_id(business_party_id):
 
     try:
         response.raise_for_status()
-    except requests.exceptions.HTTPError:
         if response.status_code == 204:
             logger.info('No case groups found for business', party_id=business_party_id)
             return []
```

Comment:
Is this (or any of the changes) going to work?  On success it returns response.json (a dictionary), on a 204, it returns a list.